src/modules/ekf2/EKF/python/ekf_derivation/derivation.py

```diff
@@ -313,6 +313,23 @@ def compute_wind_init_and_cov_from_airspeed(
     P = P.subs({sideslip: 0.0})
     wind = wind.subs({sideslip: 0.0})
     return (wind, P)
+def compute_wind_init_and_cov_from_wind_speed_and_direction(
+        wind_speed: sf.Scalar,
+        wind_direction: sf.Scalar,
+        wind_speed_var: sf.Scalar,
+        wind_direction_var: sf.Scalar
+)-> (sf.V2, sf.M22):
+    wind = sf.V2(wind_speed * sf.cos(wind_direction), wind_speed * sf.sin(wind_direction))
+    J = wind.jacobian([wind_speed, wind_direction])
+
+    R = sf.M22()
+    R[0,0] = wind_speed_var
+    R[1,1] = wind_direction_var
+
+    P = J * R * J.T
+
+    return (wind, P)
+
```

Comment:
Can you change the used variable of the jacobian to H here and in compute_wind_init_and_cov_from_airspeed()? Just to be consistent with the other derivations.